src/ipmi.c

```diff
@@ -798,9 +805,8 @@ entity_sensor_update_handler(enum ipmi_update_e op,
             sensor, sensor_discrete_event_handler, st);
 
       if (status) {
```

Comment:
brackets are not needed in this condition.

Reply:
done